README.md

```diff
@@ -30,6 +30,17 @@ for i in range(len(df_page)):
 		" PageRank is " + str(df_page['pagerank'].iloc[i]))
 ```
 
+## Getting cuGraph
+There are 3 ways to get cuGraph :
+1. [Quick start with Docker Demo Repo](#quick)

```

Comment:
I don't think they're calling it a demo repo anymore, so I would just change it to "Quick start with the RAPIDS Docker repo"

Reply:
updated in latest push